### ISSUE TITLE
vATIS Beta testen

### ISSUE THREAD

**Issue by Phil061995** [VATGER-Nav/edgg-package#263]
https://vatis.app/

Featurerequest und Bugreport auf dem verlinkten Discord Server.

**Comment by Phil061995** [comment]
@pa-ulex du scheinst hier ja fleißig am testen zu sein, wenn du magst kannst du gerne deine Profile teilen, dass wir nicht 5 Mal was erstellen :)

**Comment by capzDE** [comment]
> Ich muss mal schauen ob es meinen Knowledbase-Artikel dazu noch gibt oder nicht.

Ja: https://knowledgebase.vatsim-germany.org/books/entwurfe/page/vatis-usage-guide

**Comment by Phil061995** [comment]
Testphase ab kommender Woche für DUS und STR geplant, file wird hier im Repo verfügbar sein.

**Comment by Phil061995** [comment]
Wegen dem letzten vATIS Update gabs noch ein paar Änderungen, die die Testphase verzögert haben. Wenn jetzt nix gravierendes dazwischen kommt, starten wir die Testphase Anfang der kommenden Woche.

**Comment by Phil061995** [comment]
Update zum testen aller Airports in Langen folgt die Tage.
